src/main/kotlin/dev/reimer/tex/gradle/plugin/compiler/bibliography/Biber.kt

```diff
@@ -67,7 +67,7 @@ open class Biber : DefaultBibliographyCompiler() {
 
     private val bcfFileName: Provider<String> = jobName.map { "$it.${FileExtensions.BCF}" }
 
-    @get:InputFile
+    @get:InputDirectory
```

Comment:
I don't see why this should be changed to a directory.

Reply:
You are right, this is not required anymore. I had the compiler complaining about this, but it is not needed anymore. Will remove this